### ISSUE TITLE
Remove --static option for twisted.web

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3907]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#3907|
|Type|task|
|Created|2009-07-08 01:24:19Z|
|Branch|https://github.com/twisted/twisted/tree/static-web-3907-2|

There's a ```--static``` option in [twisted/web/tap.py twisted web](twisted/web/tap.py twisted web) that does the same ```--path```. It's marked as deprecated:

```python
    def opt_static(self, path):
        """Same as --path, this is deprecated and will be removed in a
        future release."""
        print ("WARNING: --static is deprecated and will be removed in"
               "a future release. Please use --path.")
        self.opt_path(path)
    opt_s = opt_static
```

Let's get rid of it.

<details><summary>Searchable metadata</summary>

```
trac-id__3907 3907
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_static_web_3907_2 branches/static-web-3907-2
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1247016259000000 1247016259000000
changetime__1263694257000000 1263694257000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @thijstriemstra|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

Sounds good.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [27115]) Branching to 'static-web-3907'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

(In [28008]) Branching to 'static-web-3907-2'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra commented|
|-|-|

(In [28009]) Remove --static option, refs [#3907](https://github.com/twisted/twisted/issues/3907)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra commented|
|-|-|

(In [28010]) Revert r28009, refs [#3907](https://github.com/twisted/twisted/issues/3907)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_8"></a>|@thijstriemstra commented|
|-|-|

(In [28011]) Coding standard, refs [#3907](https://github.com/twisted/twisted/issues/3907)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_9"></a>|@thijstriemstra commented|
|-|-|

(In [28012]) Remove --static option, refs [#3907](https://github.com/twisted/twisted/issues/3907)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_10"></a>|@thijstriemstra removed owner|
|-|-|

Added news file in r28013. Up for review. Sorry about r28009 :-/

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)<a name="note_11"></a>|@moshez set owner to @thijstriemstra|
|-|-|

Looks good. Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_12"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [28020]) Merge static-web-3907-2: Remove '--static' option from twistd web, that served
as an alias for the '--path' option.

Author: thijs
Reviewer: moshez
Fixes: [#3907](https://github.com/twisted/twisted/issues/3907)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
